docs/dsms_sdk/dsms_config_schema.md

```diff
@@ -14,8 +14,9 @@ This section describes the configuration properties for the DSMS Python SDK.
 | Password              | Password for connecting to the DSMS instance                                                                                                                                                                                                                                                        | `Optional[SecretStr]`| `None`                       | `password`              | Optional          | `DSMS_PASSWORD`               |
 | Token                 | JWT bearer token for connecting to the DSMS instance                                                                                                                                                                                                                                                | `Optional[SecretStr]`| `None`                       | `token`                 | Optional          | `DSMS_TOKEN`                  |
-| Ping DSMS             | Check whether the host is a DSMS instance or not.                                                                                                                                                                                                                                                   | `bool`               | `True`                       | `ping_dsms`             | Optional          | `DSMS_PING_DSMS`              |
+| Ping Backend             | Check whether the host is a DSMS instance or not.                                                                                                                                                                                                                                                   | `bool`               | `True`                       | `ping_backend`             | Optional          | `DSMS_PING_BACKEND`              |
 | Auto fetch KTypes     | Whether the KTypes of the DSMS should be fetched automatically when the session is started. They will be fetched if requested and cached in memory.                                                                                                                                                | `bool`               | `True`                       | `auto_fetch_ktypes`     | Optional          | `DSMS_AUTO_FETCH_KTYPES`      |
+| Auto refresh     | Whether objects like `KItem`, `KType` and `AppConfig` should be refreshed locally after committing them to the DSMS.                                                                                            | `bool`               | `True`                       | `auto_refresh`     | Optional          | `DSMS_AUTO_REFRESH`      |
```

Comment:
The explanation doesn't clarify exactly what it does - I'm still unsure

Reply:
The local objects in the memory might still have an outdated payload after committing. Eg. the custom properties after file upload. Hence they need to be refresh with the latest payload from the backend after a successful commit. This can be done manually in order automatically with this config field.

Reply:
e.g.
```python
item = KItem(...)
dsms.add(item)
dsms.commit()
item.refresh # in case if `auto_refresh` is disabled
```

Reply:
Ah I see. How about the following text?

_Determines whether local objects like `KItem`, `KType`, and `AppConfig` should automatically update with the latest backend data after a successful commit._

Reply:
Done